controllers/get.controllers.js

```diff
@@ -11,3 +11,15 @@ exports.getApiEndpoints = (req, res, next) => {
         res.status(200).send(endpointDescriptions)
     })
 }
+
+exports.getArticleById = (req, res, next) => {
+    const articleId = req.params.article_id
+    fetchArticleById(articleId).then((article) => {
+        res.status(200).send(article)
+    }).catch((err) => {
+        if(err === "Not found"){
```

Comment:
you should just be passing any errors (like the Promise.reject from you models) to your middleware, so I would remove this line and just invoke next with your error